osidb/serializer.py

```diff
@@ -623,6 +624,32 @@ def to_representation(self, value):
         return list(trackers)
 
 
+class FlawAcknowledgmentSerializer(
+    ACLMixinSerializer,
+    BugzillaSyncMixinSerializer,
+    IncludeExcludeFieldsMixin,
+    TrackingMixinSerializer,
+):
+    """FlawAcknowledgment serializer"""
+
+    class Meta:
+        """filter fields"""
+
+        model = FlawAcknowledgment
+        fields = (
+            ["name", "affiliation", "from_upstream", "flaw", "uuid"]
+            + ACLMixinSerializer.Meta.fields
+            + TrackingMixinSerializer.Meta.fields
+        )
+
+
+@extend_schema_serializer(exclude_fields=["updated_dt"])
```

Comment:
`flaw` should be also excluded

Reply:
Fixed in commit 3c2d2dd147380fc8924fac4b9d539f85f86f36d4 (hopefully, see discussion thread https://github.com/RedHatProductSecurity/osidb/pull/265#discussion_r1250297359 ). Thanks!